wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserService.java

```diff
@@ -161,6 +165,9 @@ public Response create(@ApiParam(value = "New user")
         }
 
         final User user = context.isUserInRole("system/admin") ? fromEntity(userDescriptor) : fromToken(token);
+        if (isValidUserName(user.getName())) {
```

Comment:
it looks odd to have isValid call that throw exception ? like there is a problem there